nginx/frontend.conf

```diff
@@ -36,54 +34,4 @@ server {
         proxy_set_header Host $host;
         proxy_set_header "X-Forwarded-Proto" "https";
     }
-}
```

Comment:
It's now in https://github.com/guardian/identity-platform/blob/master/nginx/identity.conf#L148

To setup identity related nginx you wourld run `identity-platform/nginx/setup.sh`